### DISCUSSION TITLE
Option to set the rem value for an entire tailwind config

### DISCUSSION THREAD

**Discussion by mrassili** [tailwindlabs/tailwindcss#1892] [Help]
hello

I've been using tailwindcss recently for a chrome extension, to encapsulate the styling I used shadow dom, it prevents styles from leaking in or out of its content.

The problem I found is when websites have set the font-size on `html` element, I don't have a way to override (and don't want to). Setting :host, :root' font-size doesn't work because `rem` looks for `html` element

Is there an option or workaround this, maybe telling tailwind to use a specific `rem`value to base it's sizes off?

**Comment by greghuc** [comment]
@mrassili did you find a solution to using tailwinds in your shadow dom / chrome extension? I've just hit the same problem with font-sizes, even though I used :host { all: initial }. Many thanks Greg

**Reply by mrassili** [reply]
I wrote about it here: www.freecodecamp.org/news/chrome-extension-with-parcel-tailwind

**Reply by igorlanko** [reply]
@mrassili, doesn't say anything about the issue you yourself raised though? I mean yes, shadow DOM is a solution, but font-size still bleeds in when you inline the styles.

**Comment by ghost** [comment]
You just need to overwrite the spacing from the config file based on your base size🤔.

**Comment by ghost** [comment]
#4521 
#4518 
Can you upvote those, please🌻
